### ISSUE TITLE
fetchRange on a trackable memory store crash IE10

### ISSUE THREAD

**Issue by sbrunot** [SitePen/dstore#49]
The following sample code is crashing Internet Explorer 10 (tested on Windows 7 and Windows 8, with version 0.2.0 of dstore):

``` js
require([
    "dstore/Memory",
    "dstore/Trackable"
], function (Memory, Trackable) {
    var TrackableStore = Memory.createSubclass([Trackable], {});
    var store = new TrackableStore();
    for (var i = 0; i < 100; i++) {
        store.add({label: "item " + i});
    }
    var queryResult = store.filter({});
    var tracked = queryResult;
    var tracked = queryResult.track();
    var firstRange = tracked.fetchRange({start: 0, end: 19});
    // the following is failing on IE10
    var secondRange = tracked.fetchRange({start: 20, end: 40});
});

```

**Comment by xiaohulu** [comment]
When I use IE 10.0.9200.16521, reproduce the issue, ie browser crashed.

**Comment by byg** [comment]
I had the same question in ie10,and i fixed it by adding flow code in "dstore/Trackable.js  " 127 lines:
`if(has('ie')=10){
     partialResults[start] = undefined;
}`
 but modified the source code  just a tempory way ....Hoping it will be fixed in 2.0 version.
